modes/tmtv/src/index.js

```diff
@@ -58,7 +58,7 @@ function modeFactory({ modeConfiguration }) {
       // Init Default and SR ToolGroups
       initToolGroups(toolNames, Enums, ToolGroupService, commandsManager);
 
-      const activateWindowLevel = () => {
+      const setWindowLevelActive = () => {
```

Comment:
If you are going to make this change, you should also change the CLI templates to match.  However, I have a configurable mode that mostly abstracts this type of event into "this.setDefaultToolActive()", so if you are going to change it, can you hide the details of WHICH tool gets active.